Casbin/Config/DefaultConfig.cs

```diff
@@ -159,26 +164,51 @@ private bool AddConfig(string section, string option, string value)
 
         private void Parse(string configFilePath)
         {
-            using (var sr = new StreamReader(configFilePath))
+            ParseBuffer(ParseStringInit(new StreamReader(configFilePath)));
+        }
+
+        private MemoryStream ParseStringInit(TextReader textReader)
```

Comment:
```suggestion
        private Stream RemoveComment(Stream stream)
```
Plz try to design by this method signature, and be called from the `AddStream` method